src/main/java/com/minecolonies/coremod/colony/managers/StatisticAchievementManager.java

```diff
@@ -268,6 +268,7 @@ public void triggerAchievement(@NotNull final MineColoniesAchievement achievemen
 
         this.colonyAchievements.add(achievement);*/
 
-        AchievementUtils.syncAchievements(colony);
+        //AchievementUtils.syncAchievements(colony);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)